packages/relay/src/durable-object.ts

```
@@ -0,0 +1,173 @@
+// disconnected, messages are queued in Durable Object storage
+// and delivered on reconnect.
+
+import type { RelayMessage, RelayResponse, Env } from "./types.js";
+import { sendLineReply, sendLinePush } from "./webhooks/line.js";
+import { sendTelegramMessage } from "./webhooks/telegram.js";
+import { PLATFORMS } from "./types.js";
+
+const MAX_QUEUE_SIZE = 1000;
+const QUEUE_KEY_PREFIX = "q:";
+
+export class RelayDurableObject implements DurableObject {
```

ESLint

[error] 'DurableObject' is not defined.

(no-undef)